low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1145,6 +1233,15 @@ def _parse_disk_info(self, index, date, _time, sensor, sensor_num, event, status
                         specific_info[key]).replace('\n',','))
                 except KeyError:
                     pass
+            
+            if alert_type == "fault":
+                self.faulty_resources[sensor_id] = {
+                    'status': status,
+                    'event': event,
+                    'resource': 'disk'
```

Comment:
This should be corrected as mentioned for other fru types.
Please take care.

Reply:
Done